### PR TITLE
Updating docs to explain linkOpened event

### PR DESCRIPTION
**Pull request by denis-rossati**
## Summary

- [x] Adding the `linkOpened` event
- [x] Adjusting trailing space and empty blank space issues

### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: docs/events.md [hunk 1]
```diff
@@ -63,6 +64,7 @@ There are several event types that you can record within the customer journey:
 | [`sessionAttributesChanged`](#sessionattributeschanged) | Miscellaneous | Yes           | Records session's attributes changes.
 | [`nothingChanged`](#nothingchanged)                     | Miscellaneous | Yes           | Records a period of inactivity.
 | [`eventOccurred`](#eventOccurred)                       | Miscellaneous | No            | Records a custom event.
+| ['linkOpened'](#linkOpened)                             | Miscellaneous | No            | Records a opened link.
```
Comment (amorimjuliana) [inline docs/events.md:67]: To keep the code standard, please use `Records a link open.` (similar to the `pageOpened` event).

Comment (amorimjuliana) [reply docs/events.md:67]: Also, can you please add this one right after the `postViewed` event, please? The last three events are more general events, It's better if we can keep them together.

File: docs/events.md [hunk 2]
```diff
@@ -1072,4 +1088,34 @@ croct.track('eventOccurred', {
...
+
+You should track this event when a user opens in a link.
+
+#### Properties
+
+This event supports the following properties:
+
+| Property | Type     | Required | Constraints | Description
+|----------|----------|----------|-------------|-------------------------------
+| `link`   | `string` | Yes      | A valid URL | The link that the user opened
```
Comment (amorimjuliana) [inline docs/events.md:1106]: Actually, we support URI here. So the constraint should be "A valid URI".

**Review by amorimjuliana — CHANGES_REQUESTED** [review]

File: docs/events.md [hunk 3]
```diff
@@ -60,6 +61,7 @@ There are several event types that you can record within the customer journey:
 | [`goalCompleted`](#goalcompleted)                       | Analytics     | No            | Records a goal completion.
 | [`interestShown`](#interestShown)                       | Miscellaneous | No            | Records a user has shown interest in something.
 | [`postViewed`](#postViewed)                             | Miscellaneous | No            | Records a post view.
+| ['linkOpened'](#linkOpened)                             | Miscellaneous | No            | Records a link open.
```
Comment (marcospassos) [inline docs/events.md:64]: This event records a link opening.

**Review by marcospassos — CHANGES_REQUESTED** [review]

**Pull request merged**